src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -25,6 +25,29 @@ export interface DropInfo {
     instruction: GradingInstruction;
 }
 
+/**
+ * Possible tutor feedback states upon validation from the server.
+ */
+export enum FeedbackCorrectionErrorType {
+    INCORRECT_SCORE = 'INCORRECT_SCORE',
+    UNNECESSARY_FEEDBACK = 'UNNECESSARY_FEEDBACK',
+    MISSING_GRADING_INSTRUCTION = 'MISSING_GRADING_INSTRUCTION',
+    INCORRECT_GRADING_INSTRUCTION = 'INCORRECT_GRADING_INSTRUCTION',
+}
+
+/**
+ * Wraps the information returned by the server upon validating tutor feedbacks.
+ */
+export class FeedbackCorrectionError {
+    // Corresponds to `Feedback.reference`. Reference to the assessed element.
+    public reference: string;
```

Comment:
how would such a string reference look like? I could only imaging referencing by ids? Maybe do not understand the concept here :)

Reply:
and those Id's are string not numbers?

Reply:
yes, IDs are strings as seen here: https://github.com/ls1intum/Artemis/blob/develop/src/main/java/de/tum/in/www1/artemis/domain/TextBlock.java#L86-L90 @lschlesinger